direct-auth-samples/pom.xml

```diff
@@ -89,6 +96,77 @@
             <version>2.5.14</version>
             <scope>test</scope>
         </dependency>
+
+        <!-- Used to parse the email out of the id token -->
+        <dependency>
+            <groupId>com.okta.jwt</groupId>
```

Comment:
We don't use this in the sample any more. I think you can make this test scoped if you need it there.